docs/usage/config-presets.md

```diff
@@ -42,6 +33,17 @@ Alternatively, Renovate can fetch preset files from an HTTP server.
 
 You can set a Git tag (like a SemVer) to use a specific release of your shared config.
 
+### Preset File Naming
+
+Presets are repo-hosted, and you can have one or more presets hosted per repository.
+If you omit a file name from your preset (e.g. `github>abc/foo`) then Renovate will look for a `default.json` file in the repo.
+If you wish to have an alternative file name, you need to specify it (e.g. `github>abc/foo:alternative-name.json5`).
+
+<!-- prettier-ignore -->
```

Comment:
This PR seems related to the table as well. So if you're going to drop the table, maybe that PR needs changes also?

- https://github.com/renovatebot/renovate/pull/28256